src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -612,7 +612,7 @@ type FrameworkIdentifier =
         | DotNetCoreApp DotNetCoreAppVersion.V1_0 -> [ DotNetStandard DotNetStandardVersion.V1_6 ]
         | DotNetCoreApp DotNetCoreAppVersion.V1_1 -> [ DotNetCoreApp DotNetCoreAppVersion.V1_0 ]
         | DotNetCoreApp DotNetCoreAppVersion.V2_0 -> [ DotNetCoreApp DotNetCoreAppVersion.V1_1;  DotNetStandard DotNetStandardVersion.V2_0 ]
-        | DotNetCoreApp DotNetCoreAppVersion.V2_1 -> [ DotNetCoreApp DotNetCoreAppVersion.V1_1;  DotNetStandard DotNetStandardVersion.V2_0 ]
+        | DotNetCoreApp DotNetCoreAppVersion.V2_1 -> [ DotNetCoreApp DotNetCoreAppVersion.V2_0;  DotNetStandard DotNetStandardVersion.V2_0 ]
```

Comment:
I guess we can remove dotnetstandard version here we 2.0 already supports that

Reply:
sure, let me add a test `netcoreapp2.1`+`netstandard2.0` and i'll remove it